plutus-tx/src/PlutusTx/IsData/Class.hs

```diff
@@ -159,10 +157,7 @@ instance FromData Builtins.BuiltinBLS12_381_G1_Element where
     fromBuiltinData d =
         case fromBuiltinData d of
           Nothing -> Nothing
-          Just (BI.BuiltinByteString bs) ->
-              case BLS12_381.G1.uncompress bs of
-                Haskell.Left _  -> Nothing
-                Haskell.Right g -> Just $ toBuiltin g
+          Just bs -> Just $ bls12_381_G1_uncompress bs
```

Comment:
@kwxm This was very wrong, we cannot use Haskell within Plinth and we have an actual Plinth builtin to call here anyway.

Reply:
How was this not noticed before - I guess nobody has been using it?

Reply:
Yeah, I can't think of another explanation.